modules/angsd/docounts/meta.yml

```
@@ -0,0 +1,73 @@
+name: "angsd_docounts"
+description: Sometimes we want or need the frequency of the different bases. This is what -doCounts does.
+keywords:
+  - angsd
+  - population genetics
+  - allele counts
+  - doCounts
+tools:
+  - "angsd":
+      description: "ANGSD: Analysis of next generation Sequencing Data"
+      homepage: "http://www.popgen.dk/angsd/"
+      documentation: "http://www.popgen.dk/angsd/"
+      tool_dev_url: "https://github.com/ANGSD/angsd"
+      doi: "10.1186/s12859-014-0356-4"
+      licence: "['GPL v3, MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: A list of BAM or CRAM files
+      pattern: "*.{bam,cram}"
+  - bai:
+      type: file
+      description: List of BAM/CRAM index files
+      pattern: "*.{bai,csi}"
+  - minqfile:
+      type: file
+      description: File with individual quality score thresholds
+      pattern: "*"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  #
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - depthsample:
+      type: file
+      description: Distribution of sequencing depths
+      pattern: "*.depthSample"
+  - depthglobal:
+      type: file
+      description: Distribution of sequencing depths
+      pattern: "*.depthGlobal"
+  - qs:
```

```suggestion
  - depth_sample:
      type: file
      description: Distribution of sequencing depths
      pattern: "*.depthSample"
  - depth_global:
      type: file
      description: Distribution of sequencing depths
      pattern: "*.depthGlobal"
  - qs:
```

Update channel names